packages/shared/lib/core/wallet/actions/events-handlers/handleNewOutputEvent.ts

```diff
@@ -42,6 +52,12 @@ export async function handleNewOutputEventInternal(walletId: string, payload: Ne
         !get(allWalletActivities)[walletId].find((_activity) => _activity.id === outputData.outputId)
     const isNftOutput = output.type === OutputType.Nft
 
+    const isImplicitAccountOutput =
```

Comment:
its missing to check that the address unlock condition belongs to the implicit account address, you can do so  by checking that the address unlock condition is your implicit account creation address, or by checking the type of that address